package.json

```diff
@@ -34,7 +34,7 @@
 	"dependencies": {
 		"bluebird": "^3.7.2",
 		"chalk": "^1.1.3",
-		"ecmarkdown": "NEXT",
+		"ecmarkdown": "/Users/kevin/code/ecmarkdown",
```

Comment:
ftr you can use a normal version here, and use `npm link` locally, without having to put anything odd in the branch

Reply:
The commit was accidental, but avoiding `npm link` was intentional; at least half the time I try to use it it ends up corrupting the state of my `node_modules`.

Reply:
`rm -rf node_modules && npm install` is a pretty easy fix for any node_modules issues, but k

Reply:
Sure but just not having to fix your node_modules is better